tests/utils/streams/processing/strategies/test_streaming.py

```diff
@@ -135,25 +136,27 @@ def test_collect() -> None:
 def test_message_batch() -> None:
     partition = Partition(Topic("test"), 0)
 
-    with SharedMemoryManager() as smm:
-        block = smm.SharedMemory(4096)
-        assert block.size == 4096
+    smm = SharedMemoryManager()
```

Comment:
Could not get the types to work correctly using `with` statement

Reply:
Did `smm.shutdown()` get called if there was an AssertionError anywhere here? Did the context manager handle that? I guess it's not a huge deal in tests, but still.

Reply:
Yeah, that's the part the context manager took care of automatically